_data/sites.json

```
@@ -3856,12 +3856,30 @@
         ]
     },
     
+    {
+        "name": "Disney",
+        "url": "https://support.disney.com/hc/en-gb/articles/115005418823-How-do-I-delete-my-Disney-account-",
+        "difficulty": "easy",
+        "domains": [
+            "https://www.disney.co.uk",
+            "https://www.disney.com",
+            "https://www.disney.de",
+            "https://www.disney.es",
+            "https://www.disney.fi/",
+            "https://www.disney.fr",
+            "https://www.disney.my/",
+            "https://www.disney.pt/",
+            "https://support.disney.com"
```

Actually these domains should not have the "https://", it was just an oversight from the entry below.
```suggestion
            "disney.co.uk",
            "disney.com",
            "disney.de",
            "disney.es",
            "disney.fi",
            "disney.fr",
            "disney.my",
            "disney.pt",
            "support.disney.com"
```